tests/private-active-active/versions.tf

```diff
@@ -14,7 +14,7 @@ terraform {
 
     random = {
       source  = "hashicorp/random"
-      version = "~> 3.0"
+      version = "= 3.1.0"
```

Comment:
This hard pin is to avoid the issue caused in the 3.1.1 release of the random provider. Hopefully this will be fixed before EOD today and we can revert this in this pull request

https://github.com/hashicorp/terraform-provider-random/issues/217